include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -4349,7 +4349,7 @@ namespace VectorTools
                  cross_product_rhs;
 
           // Store all normal vectors at quad points:
-          std::vector<Point<dim> > normal_vector_list(fe_face_values.get_normal_vectors());
+          std::vector<Tensor<1,dim> > normal_vector_list = fe_face_values.get_all_normal_vectors();
```

Comment:
Shouldn't this be a const reference to the normal vectors? `normal_vector_list` is not modified in the code below. Or even better, skip this variable and use fe_face_values.normal_vector(q_point) below.